packages/lib-classifier/src/plugins/tasks/experimental/volumetric/components/VolumetricTask.js

```diff
@@ -0,0 +1,90 @@
+import { Box, Text } from "grommet"
+import { Blank } from "grommet-icons"
+import { observer } from "mobx-react"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import { Markdownz } from "@zooniverse/react-components"
+import InputStatus from "../../../components/InputStatus"
+import TaskInput from "../../../components/TaskInput"
+
+// Note: ANNOTATION_COUNT will be refactored in next PR to use MobX Annotations
+const ANNOTATION_COUNT = 3
+const SVG_ARROW = "48 50, 48 15, 40 15, 50 0, 60 15, 52 15, 52 50"
+
+const StyledInstructionText = styled(Text)`
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  > *:first-child {
+    margin-top: 0;
+  }
+`
+
+const StyledToolIcon = styled.div`
+  background-color: #2d2d2d;
+  display: flex;
+  align-items: center;
+  padding-left: 15px;
+
+  &::after {
+    content: " ";
+    margin-right: 1ch;
+    white-space: pre;
+  }
+
+  > svg {
+    height: 1.5em;
+    vertical-align: bottom;
+    width: 1.5em;
+  }
+`
+
+function VolumetricTask({ task }) {
+  return (
+    <Box>
+      <StyledInstructionText as="legend" size="small">
+        <Markdownz>{task.instruction}</Markdownz>
+      </StyledInstructionText>
+
+      <TaskInput
+        checked={true}
```

Comment:
Is this hardcoded as `true` because there's only ever one active tool in any volumetric project? If that's the case please include a brief code comment here because `checked` and `index` props for TaskInput were originally designed for multi-tool drawing tasks.

Reply:
Yes - that's why. Added the comment.